### ISSUE TITLE
problem during installation of version 1.0.1 with `pip install pynbody`

### ISSUE THREAD

**Issue by elehcim** [pynbody/pynbody#575]
Hello,
many thanks for the effort of this release!
I get a problem when installing pynbody with pip in a newly created python3.8 conda environment. It seems to be related to `setup.py`.
The problem is not there if I clone repo and do `pip install .`

```
$ conda create -n p1 python
...

$ pip --version
pip 20.2.2 from /miniconda3/envs/p1/lib/python3.8/site-packages/pip (python 3.8)

$ pip install pynbody
Collecting pynbody
  Downloading pynbody-1.0.1.tar.gz (2.6 MB)
     |████████████████████████████████| 2.6 MB 2.1 MB/s 
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /miniconda3/envs/p1/bin/python /miniconda3/envs/p1/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpyagnknys
       cwd: /tmp/pip-install-wd9xpc_w/pynbody
  Complete output (18 lines):
  Traceback (most recent call last):
    File "/miniconda3/envs/p1/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
      main()
    File "/miniconda3/envs/p1/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/miniconda3/envs/p1/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-g20xqeqg/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 162, in get_requires_for_build_wheel
      return self._get_build_requires(
    File "/tmp/pip-build-env-g20xqeqg/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 143, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-g20xqeqg/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 266, in run_setup
      super(_BuildMetaLegacyBackend,
    File "/tmp/pip-build-env-g20xqeqg/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 318, in <module>
      with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
  FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-wd9xpc_w/pynbody/README.md'
  ----------------------------------------
ERROR: Command errored out with exit status 1: /miniconda3/envs/p1/bin/python /miniconda3/envs/p1/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpyagnknys Check the logs for full command output.
```

**Comment by apontzen** [comment]
Ooops, thanks for pointing this out. I think I just fixed it by pushing a 1.0.2 to PyPI - could you check and confirm?

**Comment by elehcim** [comment]
It works, thanks!
